cmd/templ/generatecmd/cmd.go

```diff
@@ -262,7 +264,11 @@ func (cmd Generate) Run(ctx context.Context) (err error) {
 				postGenerationEventsWG.Add(1)
 				if cmd.Args.Command != "" && goUpdated {
 					cmd.Log.Debug("Executing command", slog.String("command", cmd.Args.Command))
-					if _, err := run.Run(ctx, cmd.Args.Path, cmd.Args.Command); err != nil {
+					var env []string
```

Comment:
It's probably okay to just use `os.Setenv` rather than the additional parameter to the run func. 

In tests you can use `t.Setenv` https://pkg.go.dev/testing#T.Setenv